app/models/concerns/cable_ready/updatable.rb

```diff
@@ -151,12 +156,13 @@ def enrich_association_with_updates(name, option, descendants = nil)
             klass: self,
             name: name,
             options: options,
-            reflection: reflection
+            reflection: reflection,
+            debounce_time: debounce
           ))
         end
       end
 
-      def enrich_attachments_with_updates(name, option)
+      def enrich_attachments_with_updates(name, option, debounce: 0.seconds)
```

Comment:
Should the default for `debounce` here match the other method signatures? i.e.

```ruby
debounce: CableReady.config.updatable_debounce_time
```

Reply:
Oops, that slipped through…